diff --git a/TopQuarkAnalysis/Configuration/python/patRefSel_eventCleaning_cff.py b/TopQuarkAnalysis/Configuration/python/patRefSel_eventCleaning_cff.py
index fc4c64b..f6121d0 100644
--- a/TopQuarkAnalysis/Configuration/python/patRefSel_eventCleaning_cff.py
+++ b/TopQuarkAnalysis/Configuration/python/patRefSel_eventCleaning_cff.py
@@ -6,7 +6,7 @@ HBHENoiseFilter.minIsolatedNoiseSumE        = 999999.
 HBHENoiseFilter.minNumIsolatedNoiseChannels = 999999
 HBHENoiseFilter.minIsolatedNoiseSumEt       = 999999.
 
-from RecoMET.METAnalyzers.CSCHaloFilter_cfi import *
+from RecoMET.METFilters.CSCTightHaloFilter_cfi import *
 
 from RecoMET.METFilters.hcalLaserEventFilter_cfi import *
 hcalLaserEventFilter.vetoByRunEventNumber = cms.untracked.bool( False )
